Sources/Sentry/SentryThreadInspector.m

```diff
@@ -46,11 +57,17 @@ - (id)initWithStacktraceBuilder:(SentryStacktraceBuilder *)stacktraceBuilder
         // For now we can only retrieve the stack trace of the current thread.
         if (isCurrent) {
```

Comment:
>> An alternative would to look into each thread for the one that contains the main function as first frame
>
>I'm not sure if this would also work for release builds.

AFAIK, for iOS, the app entry point is a `main` function, created manually or by the compiler, does not matter the build configuration. For MacOS it may be called `start`

But I like this idea better

> SentryCrashThread mainThreadID = pthread_mach_thread_np(pthread_self());

Reply:
is the current thread calling the function.
I did not get the problem.

Reply:
Yes. But why is this a problem? 
For ANR we don't use this flag anywhere.